CHANGES.md

```diff
@@ -19,7 +19,8 @@ Apollo 2.1.0
 * [Add search key when comparing Configuration items](https://github.com/apolloconfig/apollo/pull/4459)
 * [Optimize performance of '/apps/{appId}/envs/{env}/clusters/{clusterName}/namespaces' interface queries](https://github.com/apolloconfig/apollo/pull/4473)
 * [Add a new API to load items with pagination](https://github.com/apolloconfig/apollo/pull/4468)
-* [fix(#4474):'openjdk:8-jre-alpine' potentially causing wrong number of cpu cores](https://github.com/apolloconfig/apollo/pull/4475)
+* [fix(#4474):'openjdk:8-jre-alpine' potentially causing wrong number of cpu cores](https://githb.com/apolloconfig/apollo/pull/4475)
```

Comment:
Is this a typo?
```suggestion
* [fix(#4474):'openjdk:8-jre-alpine' potentially causing wrong number of cpu cores](https://github.com/apolloconfig/apollo/pull/4475)
```

Reply:
Sorry, it should be my problem.This is my first pr，I don't know how to fix it, whether to edit this file directly or create a new pr.

Reply:
You may fix it in your local branch and then push it to upstream.